snapshots/windows/windows.go

```diff
@@ -60,8 +59,8 @@ func init() {
 const (
 	// Label to specify that we should make a scratch space for a UtilityVM.
 	uvmScratchLabel = "containerd.io/snapshot/io.microsoft.vm.storage.scratch"
-	// Label to control a containers scratch space size (sandbox.vhdx).
-	rootfsSizeLabel = "containerd.io/snapshot/io.microsoft.container.storage.rootfs.size-gb"
```

Comment:
This seems like a breaking change to anyone using the existing label.

Reply:
Sounds like you checked and we're confident no one is using this? I am still unsure if we should leave removal of the old label for 2.0. What do you think?

Reply:
> Now that there is a CRI
field to specify the size, this should just be a field that users can set
on their pod specs and don't need to concern themselves with what it eventually
gets translated to.

Note that the label still matters as there is use of containers outside of CRI.

Reply:
Yes I couldn't find a use of it from a Github search. We don't use it in any of our work either (we do for LCOW but that label is untouched). I'd be more than fine with keeping it till 2.0 and just tally up the total depending on which was specified, and I guess if *both* were then the new bytes label would take precedence

Reply:
> > Now that there is a CRI
> > field to specify the size, this should just be a field that users can set
> > on their pod specs and don't need to concern themselves with what it eventually
> > gets translated to.
> 
> Note that the label still matters as there is use of containers outside of CRI.

Very true. I think I'm in favor of leaving it for now

Reply:
Left the original label in and marked it as deprecated and to prefer a new containerd.io/snapshot.rootfs.size-bytes that was added.